src/components/MessageBody.tsx

```diff
@@ -1,11 +1,32 @@
-import { FC } from 'react';
-import ReactMarkdown from 'react-markdown';
-import remarkGfm from 'remark-gfm';
+import Highlight, { Language, defaultProps } from 'prism-react-renderer';
+import vsLight from 'prism-react-renderer/themes/vsLight';
 import remarkBreaks from 'remark-breaks';
+import remarkGfm from 'remark-gfm';
+
+import { FC, ReactElement } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { CodeProps } from 'react-markdown/lib/ast-to-react';
+
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
   messageParagraphs: {
+    '& .prism-code': {
+      fontFamily:
+        'ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace',
```

Comment:
Are those fonts available on any system? Or do we import them through e.g. Google Fonts? Similar below

Reply:
It seems that by default `ui-monospace` [is not widely supported](https://caniuse.com/extended-system-fonts), while `SF Mono` and `Menlo` are Apple-specific, `Consolas` is Microsoft-specific and `Liberation Mono` is Red Hat Linux specific (though it is available for other systems as well).

So it really depends on how consistent you want the font experience to be. If having the exact same styling on every platform is a concern, then you should ship the font (e.g. using Google Font or a custom CSS import). If using any monospace font is the only requirement, then IMO it is fine to live it as is (the specific system of the user will pick the first available font on the machine, and default to the system's `monospace` font).